src/status_im/chaos_mode/core.cljs

```diff
@@ -0,0 +1,44 @@
+(ns status-im.chaos-mode.core
+  (:require [status-im.utils.fx :as fx]
+            [re-frame.core :as re-frame]
+            [status-im.utils.utils :as utils]
+            [status-im.utils.http :as http]
+            [status-im.utils.types :as types]))
+
+(def interval-5m (* 5 60 1000))
+(defonce interval-id (atom nil))
+(def url "https://cloudflare-dns.com/dns-query?name=chaos-unicorn-day-test.status.im&type=TXT")
```

Comment:
this domain name will change, right? cc @jakubgs

Reply:
yep, most likely. I will change it before merging

Reply:
Yeah, i assume the real one we will be using on that day will be `chaos-unicorn-day.status.im`, without the `-test`.